.github/lock.yml

```diff
@@ -0,0 +1,38 @@
+# Configuration for Lock Threads - https://github.com/dessant/lock-threads-app
+
```

Comment:
This doesn't actually work on Radarr it seems, so just leave it out until we have another solution that works